src/include/miopen/conv/problem_description.hpp

```diff
@@ -190,17 +190,17 @@ struct ProblemDescription : ProblemDescriptionBase
 
     // In getters
     miopenDataType_t GetInDataType() const { return in.GetType(); }
-    std::size_t GetInBatchSize() const { return GetN5(GetSpatialDims(), in.GetLengths()); }
-    std::size_t GetBatchSize() const { return GetInBatchSize(); } // alias of GetInBatchSize()
-    std::size_t GetInChannels() const { return GetC5(GetSpatialDims(), in.GetLengths()); }
-    std::size_t GetInDepth() const { return GetD5(GetSpatialDims(), in.GetLengths()); }
-    std::size_t GetInHeight() const { return GetH5(GetSpatialDims(), in.GetLengths()); }
-    std::size_t GetInWidth() const { return GetW5(GetSpatialDims(), in.GetLengths()); }
-    std::size_t GetInBatchStride() const { return GetN5(GetSpatialDims(), in.GetStrides()); }
-    std::size_t GetInChannelStride() const { return GetC5(GetSpatialDims(), in.GetStrides()); }
-    std::size_t GetInStrideD() const { return GetD5(GetSpatialDims(), in.GetStrides()); }
-    std::size_t GetInStrideH() const { return GetH5(GetSpatialDims(), in.GetStrides()); }
-    std::size_t GetInStrideW() const { return GetW5(GetSpatialDims(), in.GetStrides()); }
+    unsigned GetInBatchSize_() const { return GetN5(GetSpatialDims(), in.GetLengths()); }
+    unsigned GetBatchSize_() const { return GetInBatchSize_(); } // alias of GetInBatchSize_()
+    unsigned GetInChannels_() const { return GetC5(GetSpatialDims(), in.GetLengths()); }
+    unsigned GetInDepth_() const { return GetD5(GetSpatialDims(), in.GetLengths()); }
+    unsigned GetInHeight_() const { return GetH5(GetSpatialDims(), in.GetLengths()); }
+    unsigned GetInWidth_() const { return GetW5(GetSpatialDims(), in.GetLengths()); }
+    unsigned GetInBatchStride_() const { return GetN5(GetSpatialDims(), in.GetStrides()); }
+    unsigned GetInChannelStride_() const { return GetC5(GetSpatialDims(), in.GetStrides()); }
+    unsigned GetInStrideD_() const { return GetD5(GetSpatialDims(), in.GetStrides()); }
+    unsigned GetInStrideH_() const { return GetH5(GetSpatialDims(), in.GetStrides()); }
+    unsigned GetInStrideW_() const { return GetW5(GetSpatialDims(), in.GetStrides()); }
```

Comment:
[Q] I am not sure this is a good change. Let's discuss.

Reply:
I would also like to know why we went to `unsigned` from `size_t` and why the names have an `_` at the end of getters.

Reply:
@amberhassaan You can find details here https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2303#issue-1839780618 and here https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2303#issuecomment-1672048340

In most places in the library `miopen::ProblemDescription` was used, its getters returned the `int` type, the getters of `conv::ProblemDescription` returned the `size_t` type. For now, it is enough for us to have `int`. `unsigned` is used because we don't need a sign.